sample-mod/versions.properties

```diff
@@ -0,0 +1,18 @@
+## suppress inspection "SpellCheckingInspection" for whole file
+##
+## Dependencies and Plugin versions with their available updates
+## Generated by $ ./gradlew refreshVersions
+## Please, don't put extra comments in that file yet, keeping them is not supported yet.
+
+plugin.fabric-loom=0.5.12
```

Comment:
This should be fine, 0.5 isnt really stable lets see how it goes.

Reply:
it worked for what i did.. it actually seemed to fix some issue where  it would not link in the sample project.. or maybe that just because it completely wiped caches